docs/pages/monitoring/monitor-your-app.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: Monitor your app
+description: Learn how to monitor the usage of your app after its release.
+---
+
+import { ContentSpotlight } from '~/ui/components/ContentSpotlight';
+import { Terminal } from '~/ui/components/Snippet';
+
+Once your app is released, you can track anonymized usage data to give you insights on how users use your app. This data includes which updates are in use, when users experience bugs, and more.
+
+## Expo insights
+
+Expo provides the [`expo-insights`](/eas-insights/introduction/) library, which tracks information related to [EAS Update](/deploy/send-over-the-air-updates/). This data includes the app version, platform, OS version, and update adoption.
+
+You can install `expo-insights` with:
+
+<Terminal cmd={['npx expo install expo-insights']} />
+
+After you install it and release production builds on the app stores, you'll be able to see additional data on your project dashboard:
+
+<ContentSpotlight
+  alt="Active user insights on the Expo dashboard."
+  src="/static/images/monitoring/monitor-your-app/expo-insights.png"
+/>
+
+You can learn more about insights with our [EAS Insights guide](/eas-insights/introduction/).
+
+## LogRocket
+
+You can get more insights with [LogRocket](https://logrocket.com). LogRocket records user sessions and identifies bugs as your users use your app. To get started, you can install the LogRocket SDK with the following command:
+
+<Terminal cmd={['npx expo install @logrocket/react-native']} />
```

Comment:
```suggestion
<Terminal cmd={['$ npx expo install @logrocket/react-native']} />
```

Reply:
Looking at the next snippet it seems setting `minSdkVersion` might be necessary in some cases. we should cover that (maybe a callout is enough to do so)?